man/8/ceph.rst

```diff
@@ -202,6 +206,26 @@ Usage::
 	ceph config-key put <key> {<val>}
 
 
+daemon
+------
+
+Submit admin-socket commands.
+
+Usage::
+
+	ceph daemon {daemon_name|socket_path} {command} ...
```

Comment:
It would be useful to add an example here like "ceph daemon osd.0 help", to clarify what we mean by daemon name, and also to give the hint that if someone wants to know about the commands they can call help.